app/src/main/java/com/example/taskwise_eventmaster/domain/repository/RoomTaskRepository.kt

```diff
@@ -59,6 +63,28 @@ class RoomTaskRepository @Inject constructor(
         }
     }
 
+    override suspend fun getTaskByEventID(eventId: Int): Task? = withContext(Dispatchers.IO) {
+        try {
```

Comment:
I can't seem to see in which case the catch block would catch a thrown exception